sw/device/lib/testing/test_framework/ottf_isrs.S

```diff
@@ -330,9 +334,13 @@ ottf_isr_exit:
   // meaning a test is run as a FreeRTOS task, where each task maintains its own
   // stack. Otherwise, the test is run on bare-metal, and there is no TCB, and
   // only a single stack/stack pointer.
-  la t0, kTestConfig  // Load the pointer to the test configuration struct.
-  lb t1, 0(t0)        // Load the first byte of the test_config struct, which
-                      // contains the `enable_concurrency` field to check.
+  la t0, kTestConfig       // Load the pointer to the test configuration struct.
+  beqz t0, .L_skip_sp_save // If the kTestConfig pointer is NULL, we are using
```

Comment:
You probably meant `L_skip_sp_restore` here?

Reply:
Ah good catch! Fixed.